specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,30 @@
+# Pricing Unit
+
+The Pricing Unit is a measurement unit specified by the provider for determining unit prices associated with the usage or the purchase of a specific resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit. PricingUnit is complementary to and describes how [QuantityInPricingUnit](#quantityinpricingunit) is measured. While PricingUnit may appear to relate to resource and/or service consumption, PricingUnit is primarily focused on pricing and cost. It is essential not to confuse PricingUnit with [UsageUnit](#usageunit), which is dedicated to tracking resource and/or service consumption and is often at a lower granularity than PricingUnit.
```

Comment:
```suggestion
The Pricing Unit is a measurement unit specified by the provider for determining unit prices associated with the usage or the purchase of a specific resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit. PricingUnit is complementary to and describes how [QuantityInPricingUnit](#quantityinpricingunit) is measured. The PricingUnit is primarily focused on pricing and cost. It should not be confused with [UsageUnit](#usageunit), which is dedicated to tracking resource and/or service consumption and is often at a lower granularity than the PricingUnit.
```

Reply:
moving suggestion to another conversation which encapsulates additional suggestions for improvements